src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,491 @@
+package strawman
+package collection.mutable
+
+import scala._
+import scala.collection.{generic, mutable}
```

Comment:
Your implementation should not be based on `scala.collection....` but on `strawman.collection.…`